app/jobs/document_upload/process_job.rb

```diff
@@ -0,0 +1,8 @@
+class DocumentUpload::ProcessJob < ApplicationJob
+  queue_as :default
+
+  def perform(id, log_file)
+    args = { id: id }
+    RakeInteractor.new(task_name: 'fromthepage:process_document_upload', args: args, log_file: log_file).call
```

Comment:
Sample usage of the wrapper I made. Right now all relevant params are task_name, args, and log_file.

We can modify this further in the future if needed.